Form/View/ImporterViewType.php

```diff
@@ -0,0 +1,140 @@
+<?php
+declare(strict_types=1);
+
+namespace EMS\CoreBundle\Form\View;
+
+use Elasticsearch\Client;
+use EMS\CommonBundle\Helper\EmsFields;
+use EMS\CoreBundle\Entity\View;
+use EMS\CoreBundle\Form\Nature\ImporterType;
+use EMS\CoreBundle\Service\FileService;
+use EMS\CoreBundle\Service\JobService;
+use Psr\Log\LoggerInterface;
+use Symfony\Bundle\FrameworkBundle\Routing\Router;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormFactory;
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+use Twig_Environment;
```

Comment:
```suggestion
use Twig\Environmentt;
```
Using the "Twig_Environment" class is deprecated since Twig version 2.7, use "Twig\Environment" instead.

Reply:
8bf12e2fe870afe3b24f0954bf5c29b19ad63e1b